### ISSUE TITLE
Promise.map

### ISSUE THREAD

**Issue by lsentkiewicz** [wix/eslint-plugin-lodash#102]
using bluebird:
```
await Promise.map(items, (item) => {...}, maxParallel))
```

ESlint reports:
`Prefer _.map over the native function.`

We can't use lodash for promises, so it shouldn't be reported as an issue.

**Comment by ganimomer** [comment]
While `Promise.map` is not the same as `someArray.map`, you can still use Lodash for arrays of promises like in your example, bluebird or otherwise.
For instance
```js
await _.map(items, item => {/* some code that returns a promise */})
```
Should work the same way. 
From what I understand from the [bluebird docs for Promise.map](http://bluebirdjs.com/docs/api/promise.map.html), the only thing you don't have is the concurrency option.

But generally speaking, Promise.map is not a native function, it's an extension that bluebird uses.
Generally speaking, ESLint rules do not take external libraries into account, and the plugin's rules don't take any *other* libraries except for Lodash into account. 
This is similar to the [known limitation of ESLint's array-callback-return rule](http://eslint.org/docs/rules/array-callback-return#known-limitations).

If you want to use bluebird's map instead of Lodash, you should probably turn off the specific rule for that line, that file, or that project.

**Comment by lsentkiewicz** [comment]
Ok, I fixed it.
```
"lodash/prefer-lodash-method": [2, {"ignoreObjects": ["Promise"]}]
```
Must be added to .eslintrc
